go/main.go

```diff
@@ -88,6 +95,10 @@ func handle(ctx context.Context, req *pb.Request) *pb.Response {
 	defer span.Finish()
 
 	time.Sleep(10 * time.Millisecond)
+	span.LogFields(
+		otLog.Int("waited.ms", 10),
+		otLog.Int("msg.length", len(req.Message)),
+	)
```

Comment:
This was more to see how they appear in the traces than anything else. Kept for that visualizing